pkg/unit_tests/runtime.go

```diff
@@ -0,0 +1,123 @@
+package unit_tests
```

Comment:
Convention is to not have any form of special characters in package names. But, we have a utils package already. Might make sense to create a package in there called "test" and place this file in there. We can rename the functions and remove the "test" portion from the name sine the imported reference will provide that context.

Reply:
I have renamed the package to "tests"

Reply:
Additionally, I have moved the tests package under "pkg/util"